docs/core/install/linux-ubuntu.md

```diff
@@ -152,6 +153,11 @@ rm packages-microsoft-prod.deb
 sudo apt update
 ```
 
+> [!TIP]
+> The previous script was written for Ubuntu and it may not work if you're using a derived distribution, such as Linux Mint. Most likely the `$repo_version` variable won't be assigned the correct value, making the URI for the `wget` command invalid. This variable maps to the specific Ubuntu version you want to get packages for, such as 22.10 or 23.04.
```

Comment:
```suggestion
> The previous script was written for Ubuntu and it might not work if you're using a derived distribution, such as Linux Mint. It's likely that the `$repo_version` variable won't be assigned the correct value, making the URI for the `wget` command invalid. This variable maps to the specific Ubuntu version you want to get packages for, such as 22.10 or 23.04.
```